gradle-plugin/build.gradle.kts

```diff
@@ -112,6 +112,11 @@ gradlePlugin {
     }
 }
 
+publishing.publications.withType<MavenPublication>().all {
+    groupId = "io.spine.chords"
+    artifactId = "spine-chords-gradle-plugin"
```

Comment:
Please review this one once again: if the prefix is added automatically, let's remove it from here.